README.rst

```diff
@@ -114,6 +116,11 @@ Documentation can be generated in several formats::
     epub : make epub
     html : make html
 
+Contributing to libdrizzle-redux
+--------------------------------
+
+Please refer to the guide for users of libraries at `Neptune Wiki`_.
```

Comment:
I think he just means that there isn't too much explanation of what Neptune is. You could do a brief introduction sentence on Neptune might be more clear.